### DISCUSSION TITLE
gdu

### DISCUSSION THREAD

**Discussion by victormihalache** [catppuccin/catppuccin#1851] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
The app is like `ncdu` but written in Go and can be themed, [see on the repo itself.](https://github.com/dundee/gdu#styling)

**Comment by Kr1ngl3** [comment]
+1

**Comment by scarcekoi** [comment]
Done: #3044

**Reply by Bryan2333** [reply]
Thanks you for making this.

**Comment by sgoudham** [comment]
Any further discussion should take place in #3044
